port/common/j9nls.c

```diff
@@ -385,6 +385,15 @@ j9nls_lookup_message(struct OMRPortLibrary *portLibrary, uintptr_t flags, uint32
 	omrthread_monitor_enter(nls->monitor);
 
 	if (!nls->catalog) {
+		/* Don't load the catalog if VM is running in an
+		 * english locale and there is a default message
+		 */
```

Comment:
Formatting issue here.

Reply:
@fjeremic Can you clarify what the formatting issue is?

Reply:
Unless GitHub's diff view is incorrect it seems the comment is misaligned. Likely a mix of spaces and tabs on line 389. I think there may be a leading space.

Reply:
Yes, the standard way to format that kind of comment is:
```c
/* some test
 * stars lining up
 */
```
which requires a single extra space to ensure the stars align.

Reply:
Just took a look at the raw file and it seems fine. Debugged a bit and found the problem on my end. The Refined GitHub addon [1] is causing the issue. Sorry for the churn! I'll open up a bug report on their end to get this fixed.

[1] https://github.com/sindresorhus/refined-github

Reply:
https://github.com/sindresorhus/refined-github/issues/2085

Reply:
> Leaving this unresolved

No worries, it's always visible here https://github.com/eclipse/omr/pull/3912/commits/75c09985f5d5dd6cfd7c9d1e4f4d56e9406a45c4